docker/Dockerfile.wifi-connect.tmpl

```diff
@@ -10,9 +10,10 @@ RUN --mount=type=cache,target=/var/cache/apt \
 
 WORKDIR /usr/src/app
 
-RUN curl -sL -o /tmp/wifi_connect.zip "$ARCHIVE_URL" && \
+RUN if [[ ! -z "$ARCHIVE_URL" ]]; then \
+    curl -sL -o /tmp/wifi_connect.zip "$ARCHIVE_URL" && \
     unzip -o /tmp/wifi_connect.zip -d /usr/src/app && \
-    rm /tmp/wifi_connect.zip
+    rm /tmp/wifi_connect.zip; fi
```

Comment:
I follow that will fix the build, but does wifi-connect work if we miss this part?

Reply:
That `RUN` statement is a one-line if-statement. I need to add a check if the `wifi-connect` archive URL is blank (for Pi 1 and Pi 2 devices), otherwise the build process will never proceed past this point.

Reply:
To add, the latest version of WiFi Connect (`4.11.1` at the time) doesn't have archive downloads for ARM v6 devices (i.e., the Raspberry Pi 1).

Reply:
What I mean is that we should not execute this Dockerfile for pi1/pi2 at all(Can we do it?). Because in this case, we have just extra building steps that don't make sense for pi1/pi2, right?

Reply:
At the moment, yes. It makes sense not to execute the Dockerfile for Pi 1 & Pi 2.
However, it might be possible to plug like USB Wi-Fi adapters into those devices.

I'll see what I can do, thanks!

Reply:
This block needs to be reverted now with the `nmcli` change above.